org.adempiere.base.process/src/org/compiere/process/CreateWindowFromTable.java

```diff
@@ -205,7 +205,7 @@ else if (table.columnExistsInDB("Name"))
 			ProcessInfo processInfo = new ProcessInfo("", SystemIDs.PROCESS_AD_TAB_CREATEFIELDS, MTab.Table_ID, tab.getAD_Tab_ID(), tab.getAD_Tab_UU());
 
 			MPInstance instance = new MPInstance(getCtx(), SystemIDs.PROCESS_AD_TAB_CREATEFIELDS, MTab.Table_ID, tab.getAD_Tab_ID(), tab.getAD_Tab_UU());
-			instance.saveEx();
+			instance.saveEx(get_TrxName());

```

Comment:
This is inconsistent with current iDempiere practise and there's an obvious issue - if there's any error with the process, the process audit is loss since trx will be rollback.

Reply:
I think the right fix is to copy the `if (I_AD_PInstance.Table_Name.equals(p_info.getTableName()))` check from checkRecordIDCrossTenant to checkRecordUUCrossTenant

Reply:
Thanks  @hengsin - you're totally right - I missed that part